### DISCUSSION TITLE
Unexpected mDNS name "OpenThread BorderRouter #3991"

### DISCUSSION THREAD

**Discussion by olalonde** [openthread/openthread#8678] [Q&A] [answered]
I followed the guide on installing OpenThread Border Router using Raspberry Pi + NRF52840 dongle. Everything seems to work, including ot-web, up until the [mDNS step](https://openthread.io/guides/border-router/mdns-discovery).

This doesn't work:

```console
dns-sd -L "OpenThread BorderRouter" _meshcop._udp local
```

But this does (found by running `avahi-browse -a` on my rpi):

```console
dns-sd -L "OpenThread BorderRouter #3991" _meshcop._udp local
```

I'm not sure where that "#3991" comes from as I didn't change it in the CMakeLists.txt

**Comment by visuphi** [comment, marked as answer]
I guess it's generated from the extended address

https://github.com/openthread/ot-br-posix/blob/7442a3f8fec3286a41d4e2785ac8d115abde7981/src/border_agent/border_agent.cpp#L495-L505

**Reply by jwhui** [reply]
@olalonde , thanks for raising this issue.

@visuphi , is correct, with https://github.com/openthread/ot-br-posix/pull/1370 we include the extended address bytes to reduce service instance name conflicts.

I submitted https://github.com/openthread/ot-docs/pull/109 to update the docs.
